src/TurbulenceConvection/EDMF_functions.jl

```diff
@@ -778,7 +778,7 @@ function compute_up_tendencies!(
         @. tends_w +=
             w * I0f(entr_w) * (w_en - w_up) +
             I0f(buoy) +
-            nh_pressure / (ρ_f * Iaf(max(a_up, a_min)))
+            nh_pressure * Iaf(a_up) / (ρ_f * Iaf(max(a_up, a_min)))
```

Comment:
Did you try just getting rid of `a_up` here?

Reply:
yes, that related to the comment in #918

Reply:
Then how about having the max(a_up, a_min) in both nominator and denominator? I'm just guessing wildly here

Reply:
the main issue is not that I multiply and divide by `a_up` is that multiplying inside `compute_nh_pressure` is different than multiplying in the equation ....